data/brands/shop/car.json

```diff
@@ -617,6 +617,19 @@
         "shop": "car"
       }
     },
+    {
+      "displayName": "Tata",
+      "id": "",
+      "locationSet": {"include": ["001"]},
```

Comment:
You can make the location more precise by adding locations here

`"locationSet": {"include": ["in"]},|